From 97187f645177827cb38f6ff9566e1a1ac8f44222 Mon Sep 17 00:00:00 2001
From: "haotian.qu" <haotian.qu@amlogic.com>
Date: Thu, 23 Nov 2023 21:21:06 +0800
Subject: [PATCH 4/4] sm1: Modify cpu freq to 1200 [1/1]

PD#SWPL-147410

Problem:
SM1 burn failure.

Solution:
Modify cpu freq to 1200.

Verify:
local PB

Signed-off-by: haotian.qu <haotian.qu@amlogic.com>
Change-Id: Ib410892420716ffc62c71d977c158ca7057b4515
---
 board/amlogic/configs/g12a_u200_debian_v1.h | 4 ++--
 board/amlogic/configs/g12b_w400_debian_v1.h | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/board/amlogic/configs/g12a_u200_debian_v1.h b/board/amlogic/configs/g12a_u200_debian_v1.h
index 9beda9d8bee..59318291da4 100644
--- a/board/amlogic/configs/g12a_u200_debian_v1.h
+++ b/board/amlogic/configs/g12a_u200_debian_v1.h
@@ -204,7 +204,7 @@
 		"console=tty0 console=ttyS0,115200 no_console_suspend "\
 		"earlycon=aml-uart,0xff803000 ramoops.pstore_en=1 "\
 		"ramoops.record_size=0x8000 ramoops.console_size=0x4000 "\
-		"scsi_mod.scan=async xhci_hcd.quirks=0x800000 gamma=0 boot_source=emmc "\
+		"scsi_mod.scan=async xhci_hcd.quirks=0x800000 gamma=0 "\
 		"\0"\
 	"upgrade_check="\
 		"echo upgrade_step=${upgrade_step}; "\
@@ -467,7 +467,7 @@
 #define CONFIG_SYS_BOOTM_LEN (64 << 20) /* Increase max gunzip size*/
 
 /* cpu */
-#define CONFIG_CPU_CLK	1920 //MHz. Range: 360-2000, should be multiple of 24
+#define CONFIG_CPU_CLK	1200 //MHz. Range: 360-2000, should be multiple of 24
 
 /* ATTENTION */
 /* DDR configs move to board/amlogic/[board]/firmware/timing.c */
diff --git a/board/amlogic/configs/g12b_w400_debian_v1.h b/board/amlogic/configs/g12b_w400_debian_v1.h
index efa713142ef..48db135584e 100644
--- a/board/amlogic/configs/g12b_w400_debian_v1.h
+++ b/board/amlogic/configs/g12b_w400_debian_v1.h
@@ -186,7 +186,7 @@
 			"console=tty0 console=ttyS0,115200 no_console_suspend " \
 			"earlycon=aml-uart,0xff803000 ramoops.pstore_en=1 " \
 			"ramoops.record_size=0x8000 ramoops.console_size=0x4000 "\
-			"scsi_mod.scan=async xhci_hcd.quirks=0x800000 gamma=0 boot_source=emmc "\
+			"scsi_mod.scan=async xhci_hcd.quirks=0x800000 gamma=0 "\
 			"\0"\
 		"upgrade_check="\
 			"echo upgrade_step=${upgrade_step}; "\
-- 
2.25.1

